packages/config/config/devices/0x0109/za1501jp-5.json

```diff
@@ -0,0 +1,43 @@
+// Vision Security ZA1501JP-5
+// Arrival Sensor
+{
+	"manufacturer": "Vision Security",
+	"manufacturerId": "0x0109",
+	"label": "ZA1501JP-5",
+	"description": "Arrival Sensor",
+	"devices": [
+		{
+			"productType": "0x202a",
+			"productId": "0x2a01",
+			"zwaveAllianceId": [
+				3196,
+				3376
+			]
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Synchronic Time Interval",
+			"description": "Value is multiplied by the basic interval time of 16 seconds (2 = 32 seconds)",
+			"valueSize": 1,
+			"unit": "seconds",
```

Comment:
Change the unit to be "16 seconds." Then you can lose that from the description. What is this, though? Can you try to find out and change the label (preferred) or add a description?

Reply:
I couldn't find many details about what this actually does. The best I could find is: 
>This is the definition how long ZA1501-5 will contact Gateway for having a synchronic information. The basic interval time is 16 seconds, the synchronic time will be the basic interval time by Parameter 1’s value.